webpack/electron.prod.js

```diff
@@ -49,6 +50,18 @@ module.exports = {
   plugins: [
     new webpack.DefinePlugin({
       "__ELECTRON_ENV": JSON.stringify("main")
+    }),
+
+    // Ledger can't find these apparently maybe due to runtime expansion of
+    // __dirname from the dist folder.
+    //
+    // TODO: Moving them is not a great answer. Figure out why it can't find
+    // these naturally and fix it somehow.
+    new CopyWebpackPlugin({
+      patterns: [
+        { from: path.join(__dirname, "../node_modules/usb/prebuilds"), to: "prebuilds"},
+        { from: path.join(__dirname, "../node_modules/node-hid/build/Release/HID_hidraw.node"), to: path.join(__dirname, "../build")}
+      ]
```

Comment:
Not sure about this one but was the only way I could figure out to make the transport work. Maybe I didnt word this great, not the `dist` folder I guess but the build folder or `app/`